presto-orc/src/main/java/com/facebook/presto/orc/ColumnGroupReader.java

```diff
@@ -569,14 +568,14 @@ private void initMap(int base, int size, int[] map)
         }
     }
 
-    int[] copyMap(int[][] maps, int channelIdx, int numRows, int[] map)
+    private int[] copyMap(int[][] maps, int channelIdx, int numRows, int[] map)
```

Comment:
nit: channelIdx -> channelIndex
nit: numRows can be renamed to something like "rows" or "rowCount"